src/routes/(app)/calendar/+page.server.ts

```diff
@@ -1,3 +1,57 @@
-export const load = () => {
-	return {}
-}
\ No newline at end of file
+import { prisma } from '$lib/server/prismaConnection';
+import { verifySession } from '$lib/server/verifySession';
+
+enum Filter {
+	CLUB = 'club',
+	ORG = 'org'
+}
+
+export const load = async ({ cookies, url }) => {
+	const user = await verifySession(cookies.get('session'));
+	let activeFilter: Filter = Filter.CLUB;
+
+	if (url.searchParams.has('filter')) {
+		if (url.searchParams.get('filter') == 'club') {
```

Comment:
instead of using if statements (if other filters get added), it may be a better idea to dynamically resolve these from the `Filter` enum

Reply:
Hey, not entirely sure what you mean by this, if possible, can you elaborate?

Reply:
[An enum is some immutable-ish object of sorts with callable values](https://www.typescriptlang.org/play?#code/KYOwrgtgBAou0G8BQVVQBpQLxQEQA8BaANwEMAbMYXAGhTQC1s8BPEiq3JAXySQGMA9iADOg8sAB05QQHMAFHEgBKJEA), so you can reverse its keys and values and look up the keys by the value (the value in this case is `url.searchParams.get('filter')`

```ts
enum Enum {
    X = "x-value",
    Z = "y-value"
}

console.log(Enum)
```